src/onelogin/saml2/idp_metadata_parser.py

```diff
@@ -108,9 +126,12 @@ def parse(idp_metadata):
 
                     if idp_entity_id is not None:
                         data['idp']['entityId'] = idp_entity_id
+
                     if idp_sso_url is not None:
                         data['idp']['singleSignOnService'] = {}
                         data['idp']['singleSignOnService']['url'] = idp_sso_url
+                        data['idp']['singleSignOnService']['binding'] = preferred_sso_binding
```

Comment:
No, because now we use the preferred binding for parsing the XML:
```
                    sso_nodes = OneLogin_Saml2_XML.query(
                        idp_descriptor_node,
                        "./md:SingleSignOnService[@Binding='%s']" % preferred_sso_binding
                        )
```

That is, `if len(sso_nodes) > 0:` (logic like before), it's guaranteed to have the binding as specified by `preferred_sso_binding`.